rasa/data.py

```diff
@@ -0,0 +1,65 @@
+import io
+import json
+import os
+import shutil
+import tempfile
+from typing import Tuple, List, Text
+import re
+
+
+def get_core_directory(directory: Text) -> Text:
+    core_files, _ = _get_core_nlu_files(directory)
+    return _copy_files_to_new_dir(core_files)
+
+
+def get_nlu_directory(directory: Text) -> Text:
+    _, nlu_files = _get_core_nlu_files(directory)
+    return _copy_files_to_new_dir(nlu_files)
+
+
+def get_core_nlu_directories(directory: Text) -> Tuple[Text, Text]:
+    story_files, nlu_data_files = _get_core_nlu_files(directory)
```

Comment:
let's make it consistent (e.g. either call it `core_files` like in L11 or `story_files`)